dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/GitHubUtil.java

```diff
@@ -29,6 +29,7 @@
 public class GitHubUtil {
     private static final Logger log = LoggerFactory.getLogger(GitHubUtil.class);
     public static final String NO_BRANCH_WARN_FORMAT = "Couldn't find branch `%s` in repo `%s`. Waiting a second...";
+    public static final String PR_INVALID_CODE = "The PR head has invalid data.";
```

Comment:
I still don't think a user or admin will understand this error message. Can we hint at what needs to be done to fix the PR?